src/client/apps/edit/components/content/sections/images/index.tsx

```diff
@@ -232,32 +237,38 @@ const SectionImagesContainer = styled.section.attrs<{
   `};
 `
 
-const SectionImagesList = styled(Flex)`
+const SectionImagesList = styled(Flex).attrs<{
+  isEditing?: boolean
+  isWrapping?: boolean
+}>({})`
   position: relative;
-  z-index: -1;
+  z-index: ${props => (props.isEditing ? "2" : "-1")};
+
+  ${DragContainer} {
```

Comment:
Why not set the styles on `DragContainer` itself, in general it's good practice to encapsulate all styling/logic for a given component within it's definition & it's easier to reason about them.